markdown/report.Rmd

```diff
@@ -0,0 +1,416 @@
+
+<!-- Check if should use four or five # here -->
+#### Age and Gender Chart HERE
+
+1.	Figures for 2018/19 are incomplete and will be revised in October 2019 publication.  See [Appendix 2](#Appendix2) for more information.
+
+### Deprivation
+The percentage of time spent at home or in a community setting in the last six months of life by deprivation is shown in Figure 4 (Table 4 in the [data tables](LINK)). Deprivation category has been assigned using the Scottish Index of Multiple Deprivation. At a Scotland level approximately 20% of the population are within each deprivation quintile with deprivation category 1 relating to the most deprived areas and deprivation category 5 to the least deprived areas. See the [Glossary](#Glossary) for further information on Deprivation. 
+
+#### Deprivation chart HERE
+
+1.	Figures for 2018/19 are incomplete and will be revised in October 2019 publication.  See [Appendix 2](#Appendix2) for more information.
+2.	Deprivation 2016 quintiles have been used.
+
+Figure 4 shows that there is little variation across the different categories of deprivation. Deprivation ranges from 88.5% (deprivation category 2) to 89.6% (deprivation category 1). 
+The trend over time by deprivation category is presented in Figure A1.3 (see [List of Tables](LINK) or [Appendix A1](#AppendixA1)). All deprivation categories show a similar trend over time, with a slight increase in the percentage of last six months spent at home or in the community for individuals in the last two or three years. For each year in the trend (2010/11 to 2018/19) there is little variation across the different deprivation categories.
+
+### Urban Rural Classification
+The Scottish Government 6-fold [urban rural classification](http://www.gov.scot/Topics/Statistics/About/Methodology/UrbanRuralClassification) has been used to present information on whether people live in Urban or Rural areas and if there is any difference in the amount of time spent at home or in a community setting in the six months prior to death (Figure 5).
+
+#### UR Chart HERE
+
+1.	Figures for 2018/19 are incomplete and will be revised in October 2019 publication. See [Appendix 2](#Appendix2) for more information.
+
+The percentage of the last six months of life spent at home or in the community ranged from 89.0% for residents in large urban areas to 90.3% for residents from remote rural areas (Figure 5; Table 5 in [data tables](LINK)). Individuals living in remote rural areas spent, on average, approximately two extra days at home or in a community setting in the last six months of life compared to those living in large urban areas.
+
+Trend data on the percentage of the last six months of life spent at home or in a community setting by Urban/Rural classification are presented in Figure A1.4 (see [List of Tables](LINK) or [Appendix A1](LINK)).
+
+
+
+####### Glossary
```

Comment:
Change it to two hashtags ## after using the updated "NATIONAL_STATS_REPORT_TEMPLATE.docx"